microprofile/5.0/config/3.0/22.0.0.1-beta-TCKResults.adoc

```diff
@@ -0,0 +1,38 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Config 3.0.
+
+== Open Liberty 22.0.0.1-beta - MicroProfile Config 3.0 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/beta/openliberty-runtime/22.0.0.1-beta/openliberty-runtime-22.0.0.1-beta.zip[Open Liberty 22.0.0.1-beta]
+
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html[MicroProfile Config 3.0 RC5]
+
+* Public URL of TCK Results Summary:
++
+link:22.0.0.1-beta-TCKResults.html[TCK results summary]
+
+* Java runtime used to run the implementation:
++
+Java 8 and 11
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+RHEL 8
+
+Java 8 Test results:
+
+[source,xml]
+----
+----
+
+Java 11 Test results:
+
```

Comment:
Please add jdk level

Reply:
please do so for all of the tck results

Reply:
The file has been updated now

Reply:
We also need JDK for Java 11 similar to what you have done for Java 8